### DISCUSSION TITLE
[WASM] Status of RichEditBox support?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#11244] [Q&A] [answered]
I liked to understand where we're at regarding (potential) support for RichEditBox. I'm primarily interested in WASM.

I'm aware [of this](https://github.com/unoplatform/uno/issues/3848) and also glanced into the decompiled source of RichEditBox to understand that there might be native Windows components involved, which would render supporting RichEditBox a serious endeavour.

Nevertheleass it would be great to get an idea about UNO's thoughts on that subject:
- likely not going to happen in foreseeable future due to obstacles mentioned above
- are there any plans/thoughts on alternate options?
- else?

Thanks in advance

**Comment by francoistanguay** [comment, marked as answer]
No short term plans. We believe that for controls like this, it is preferrable to use a native (and proven) control per platform. This is the beauty of Uno, you're never locked in and can always fallback to native as needed.

**Reply by DierkDroth** [reply]
Thanks for your feedback @francoistanguay
